test/server.js

```diff
@@ -7,16 +7,45 @@ let app = express();
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
+let hooks = {};
 const config = {
   db: {
     host: 'localhost',
     port: 5432,
     name: 'sensorthingstest',
     user: 'postgres',
-    pass: '12345678'
+    pass: '12345678',
+    hooks: {
+        beforeCreate: () => {
+          hooks.beforeCreate = true
+        },
+        afterCreate: () => {
+          hooks.afterCreate = true
+        },
+        beforeDestroy: () => {
+          hooks.beforeDestroy = true
+        },
+        afterDestroy: () => {
+          hooks.afterDestroy = true
+        },
+        beforeUpdate: () => {
+          hooks.beforeUpdate = true
+        },
+        afterUpdate: () => {
+          hooks.afterUpdate = true
+        }
+    }
   }
-};
+}
 
 app.use('/', SensorThings(config));
 
-exports = module.exports = app;
+exports.app = app;
```

Comment:
use `export default app`. This way you don't need to do `import { app } from './server';` everywhere.